clone-replace.php

```diff
@@ -143,4 +145,62 @@ function cr_the_status( $post ) {
 		}
 	}
 
+	function cr_maybe_enqueue_gutenberg_script() {
```

Comment:
This function will always enqueue the script, so I would recommend using our typical standard for callbacks:
```suggestion
	function cr_action_enqueue_block_editor_assets() {
```